duckduckgo/Android
# Create Settings file that applies our Style Guide

Task/Issue URL:  https://app.asana.com/0/414730916066338/1169742290306273

**Description**:
Our style guide asks developers to change some settings in the IDE. This can be simplified, but creating a file that can be imported to the IDE and applies all the changes we want. 

**Steps to test this PR**:
1. Open Android Studio Settings / Code Style and change the Hard Wrap at 100 columns
2. Follow the Instructions in the Style Guide to import the Settings File 
3. Restart Android Studio
4. Check that the Hard Wrap is now at 150 columns
